### ISSUE TITLE
Implement MySqlConnection.ChangeDatabase

### ISSUE THREAD

**Issue by xyting** [mysql-net/MySqlConnector#201]
Currently, I user this package for test, I will dynamically change the database name, but I found the `ChangeDatabase(string)` method throw NotImplemtedException

```csharp
public override void ChangeDatabase(string databaseName)
{
	throw new NotImplementedException();
}
```

**Comment by bgrainger** [comment]
This should be implemented by sending a [`COM_INIT_DB`](https://dev.mysql.com/doc/internals/en/com-init-db.html) packet.

**Comment by bgrainger** [comment]
Implemented in [0.15.0](https://www.nuget.org/packages/MySqlConnector/0.15.0).

**Comment by xyting** [comment]
so quickly, thanks! good job
